ArduCopter/version.cpp

```diff
@@ -0,0 +1,16 @@
+#include "version.h"
+
+#include "GCS_MAVLink/GCS.h" // for FIRMWARE_VERSION_TYPE_DEV and friends
```

Comment:
these comments only get outdated with time. Could be removed, just include what's needed.